code/modules/vore/eating/simple_animal_vr.dm

```diff
@@ -19,6 +19,9 @@
 		to_chat(src,"<span class='warning'>Put your hands to good use instead!</span>")
 		return
 	feed_grabbed_to_self(src,T)
+	
+	if(vore_animation)//This affects all bolt action and shotguns. 
+		flick("[icon_state]-eating", src)//This plays any pumping
```

Comment:
I suggest abstracting this further, into `if(vore_animation) handle_eating_anim()` and just have the generic flick there as the prototype. Then if a mob wants to do silly things like have 7 different animations for different methods of vore or whatever then that's entirely doable.